### ISSUE TITLE
[Android]PlatformView with transparent background on native side produces wrong view

### ISSUE THREAD

**Issue by joelpaulkoch** [flutter/flutter#106543]
## Steps to Reproduce

1. Create a new flutter project for android
2. Follow this tutorial to add a native android view: https://docs.flutter.dev/development/platform-integration/android/platform-views
3. On the platform side in NativeView.kt: change `textView.setBackgroundColor(Color.rgb(255, 255, 255))` to `textView.setBackgroundColor(Color.TRANSPARENT)` (or Color.argb(...) with an alpha different from 255)
4. Run the app

**Expected results:** 
The platform view is shown with correct content and transparent background.

**Actual results:** 
The platform view is shown with the content repeated many times and partially overlapping.

<details>
<summary>Code sample</summary>

repo to reproduce: https://github.com/joelpaulkoch/platformview

The critical part is to set the background color to transparent.

```
internal class NativeView(context: Context, id: Int, creationParams: Map<String?, Any?>?) : PlatformView {
    private val textView: TextView

    override fun getView(): View {
        return textView
    }

    override fun dispose() {}

    init {
        textView = TextView(context)
        textView.textSize = 72f
        textView.setBackgroundColor(Color.TRANSPARENT)
        textView.text = "platform view test"
    }
}
```

</details>

<details>
  <summary>Logs</summary>

```
[ +108 ms] executing: uname -m
[  +53 ms] Exit code 0 from: uname -m
[        ] x86_64
[  +12 ms] executing: [/home/joel/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +15 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] cd41fdd495f6944ecd3506c21e94c6567b073278
[   +1 ms] executing: [/home/joel/flutter/] git tag --points-at cd41fdd495f6944ecd3506c21e94c6567b073278
[ +166 ms] Exit code 0 from: git tag --points-at cd41fdd495f6944ecd3506c21e94c6567b073278
[        ] 3.0.2
[  +15 ms] executing: [/home/joel/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[   +9 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [/home/joel/flutter/] git ls-remote --get-url origin
[  +10 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +136 ms] executing: [/home/joel/flutter/] git rev-parse --abbrev-ref HEAD
[   +9 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +109 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +6 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +85 ms] executing: /home/joel/Android/Sdk/platform-tools/adb devices -l
[  +81 ms] List of devices attached
           BH9007JH56             device usb:3-3 product:F5121 model:F5121 device:F5121 transport_id:1
[  +15 ms] /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell getprop
[  +84 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +221 ms] Skipping pub get: version match.
[ +137 ms] Generating /home/joel/source/platformview/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java
[  +54 ms] ro.hardware = qcom
[  +63 ms] Initializing file store
[  +20 ms] Skipping target: gen_localizations
[  +14 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents: /home/joel/source/platformview/.dart_tool/package_config_subset}
[  +43 ms] gen_dart_plugin_registrant: Complete
[   +1 ms] Skipping target: _composite
[   +2 ms] complete
[  +11 ms] Launching lib/main.dart on F5121 in debug mode...
[   +8 ms] /home/joel/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev /home/joel/flutter/bin/cache/dart-sdk/bin/snapshots/frontend_server.dart.snapshot --sdk-root
/home/joel/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill
/tmp/flutter_tools.YOYVIQ/flutter_tool.HCVQSY/app.dill --packages /home/joel/source/platformview/.dart_tool/package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation
--filesystem-scheme org-dartlang-root --initialize-from-dill build/c075001b96339384a97db4862b8ab8db.cache.dill.track.dill --enable-experiment=alternative-invalidation-strategy
[  +23 ms] executing: /home/joel/Android/Sdk/build-tools/31.0.0/aapt dump xmltree /home/joel/source/platformview/build/app/outputs/flutter-apk/app.apk AndroidManifest.xml
[  +28 ms] Exit code 0 from: /home/joel/Android/Sdk/build-tools/31.0.0/aapt dump xmltree /home/joel/source/platformview/build/app/outputs/flutter-apk/app.apk AndroidManifest.xml
[        ] N: android=http://schemas.android.com/apk/res/android
             E: manifest (line=2)
               A: android:versionCode(0x0101021b)=(type 0x10)0x1
               A: android:versionName(0x0101021c)="1.0.0" (Raw: "1.0.0")
               A: android:compileSdkVersion(0x01010572)=(type 0x10)0x1f
               A: android:compileSdkVersionCodename(0x01010573)="12" (Raw: "12")
               A: package="com.example.platformview" (Raw: "com.example.platformview")
               A: platformBuildVersionCode=(type 0x10)0x1f
               A: platformBuildVersionName=(type 0x10)0xc
               E: uses-sdk (line=7)
                 A: android:minSdkVersion(0x0101020c)=(type 0x10)0x13
                 A: android:targetSdkVersion(0x01010270)=(type 0x10)0x1f
               E: uses-permission (line=15)
                 A: android:name(0x01010003)="android.permission.INTERNET" (Raw: "android.permission.INTERNET")
               E: application (line=17)
                 A: android:label(0x01010001)="platformview" (Raw: "platformview")
                 A: android:icon(0x01010002)=@0x7f080000
                 A: android:name(0x01010003)="android.app.Application" (Raw: "android.app.Application")
                 A: android:debuggable(0x0101000f)=(type 0x12)0xffffffff
                 A: android:appComponentFactory(0x0101057a)="androidx.core.app.CoreComponentFactory" (Raw: "androidx.core.app.CoreComponentFactory")
                 E: activity (line=23)
                   A: android:theme(0x01010000)=@0x7f0a0000
                   A: android:name(0x01010003)="com.example.platformview.MainActivity" (Raw: "com.example.platformview.MainActivity")
                   A: android:exported(0x01010010)=(type 0x12)0xffffffff
                   A: android:launchMode(0x0101001d)=(type 0x10)0x1
                   A: android:configChanges(0x0101001f)=(type 0x11)0x40003fb4
                   A: android:windowSoftInputMode(0x0101022b)=(type 0x11)0x10
                   A: android:hardwareAccelerated(0x010102d3)=(type 0x12)0xffffffff
                   E: meta-data (line=38)
                     A: android:name(0x01010003)="io.flutter.embedding.android.NormalTheme" (Raw: "io.flutter.embedding.android.NormalTheme")
                     A: android:resource(0x01010025)=@0x7f0a0001
                   E: intent-filter (line=42)
                     E: action (line=43)
                       A: android:name(0x01010003)="android.intent.action.MAIN" (Raw: "android.intent.action.MAIN")
                     E: category (line=45)
                       A: android:name(0x01010003)="android.intent.category.LAUNCHER" (Raw: "android.intent.category.LAUNCHER")
                 E: meta-data (line=52)
                   A: android:name(0x01010003)="flutterEmbedding" (Raw: "flutterEmbedding")
                   A: android:value(0x01010024)=(type 0x10)0x2
                 E: uses-library (line=56)
                   A: android:name(0x01010003)="androidx.window.extensions" (Raw: "androidx.window.extensions")
                   A: android:required(0x0101028e)=(type 0x12)0x0
                 E: uses-library (line=59)
                   A: android:name(0x01010003)="androidx.window.sidecar" (Raw: "androidx.window.sidecar")
                   A: android:required(0x0101028e)=(type 0x12)0x0
[  +16 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell -x logcat -v time -t 1
[  +13 ms] <- compile package:platformview/main.dart
[  +71 ms] --------- beginning of main
           06-24 10:20:47.930 I/WifiHAL (  556): Got channel list with 15 channels
[  +22 ms] executing: /home/joel/Android/Sdk/platform-tools/adb version
[  +17 ms] Android Debug Bridge version 1.0.41
           Version 33.0.2-8557947
           Installed as /home/joel/Android/Sdk/platform-tools/adb
[   +4 ms] executing: /home/joel/Android/Sdk/platform-tools/adb start-server
[  +36 ms] Building APK
[  +39 ms] Running Gradle task 'assembleDebug'...
[   +9 ms] Using gradle from /home/joel/source/platformview/android/gradlew.
[ +110 ms] executing: /home/joel/android-studio/jre/bin/java -version
[ +459 ms] Exit code 0 from: /home/joel/android-studio/jre/bin/java -version
[        ] openjdk version "11.0.12" 2021-07-20
           OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
           OpenJDK 64-Bit Server VM (build 11.0.12+0-b1504.28-7817840, mixed mode)
[   +3 ms] executing: [/home/joel/source/platformview/android/] /home/joel/source/platformview/android/gradlew -Pverbose=true -Ptarget-platform=android-arm64 -Ptarget=/home/joel/source/platformview/lib/main.dart
-Pbase-application-name=android.app.Application -Pdart-defines=RkxVVFRFUl9XRUJfQVVUT19ERVRFQ1Q9dHJ1ZQ== -Pdart-obfuscation=false -Ptrack-widget-creation=true -Ptree-shake-icons=false -Pfilesystem-scheme=org-dartlang-root
assembleDebug
[+1609 ms] Welcome to Gradle 7.4!
[        ] Here are the highlights of this release:
[        ]  - Aggregated test and JaCoCo reports
[        ]  - Marking additional test source directories as tests in IntelliJ
[        ]  - Support for Adoptium JDKs in Java toolchains
[        ] For more details see https://docs.gradle.org/7.4/release-notes.html
[+4381 ms] > Task :app:preBuild UP-TO-DATE
[        ] > Task :app:preDebugBuild UP-TO-DATE
[   +9 ms] > Task :app:mergeDebugNativeDebugMetadata NO-SOURCE
[+23882 ms] > Task :app:compileFlutterBuildDebug
[        ] > Task :app:packLibsflutterBuildDebug UP-TO-DATE
[  +98 ms] > Task :app:compileDebugAidl NO-SOURCE
[        ] > Task :app:compileDebugRenderscript NO-SOURCE
[        ] > Task :app:generateDebugBuildConfig UP-TO-DATE
[  +99 ms] > Task :app:checkDebugAarMetadata UP-TO-DATE
[        ] > Task :app:cleanMergeDebugAssets
[  +99 ms] > Task :app:mergeDebugShaders UP-TO-DATE
[        ] > Task :app:compileDebugShaders NO-SOURCE
[        ] > Task :app:generateDebugAssets UP-TO-DATE
[        ] > Task :app:mergeDebugAssets
[ +299 ms] > Task :app:copyFlutterAssetsDebug
[        ] > Task :app:generateDebugResValues UP-TO-DATE
[        ] > Task :app:generateDebugResources UP-TO-DATE
[        ] > Task :app:mergeDebugResources UP-TO-DATE
[        ] > Task :app:createDebugCompatibleScreenManifests UP-TO-DATE
[        ] > Task :app:extractDeepLinksDebug UP-TO-DATE
[  +98 ms] > Task :app:processDebugMainManifest UP-TO-DATE
[        ] > Task :app:processDebugManifest UP-TO-DATE
[        ] > Task :app:processDebugManifestForPackage UP-TO-DATE
[  +99 ms] > Task :app:processDebugResources UP-TO-DATE
[ +100 ms] > Task :app:compileDebugKotlin UP-TO-DATE
[        ] > Task :app:javaPreCompileDebug UP-TO-DATE
[  +99 ms] > Task :app:compileDebugJavaWithJavac UP-TO-DATE
[ +300 ms] > Task :app:processDebugJavaRes NO-SOURCE
[  +99 ms] > Task :app:mergeDebugJavaResource UP-TO-DATE
[        ] > Task :app:checkDebugDuplicateClasses UP-TO-DATE
[ +301 ms] > Task :app:dexBuilderDebug UP-TO-DATE
[        ] > Task :app:desugarDebugFileDependencies UP-TO-DATE
[  +97 ms] > Task :app:mergeExtDexDebug UP-TO-DATE
[        ] > Task :app:mergeDexDebug UP-TO-DATE
[        ] > Task :app:mergeDebugJniLibFolders UP-TO-DATE
[        ] > Task :app:mergeDebugNativeLibs UP-TO-DATE
[        ] > Task :app:stripDebugDebugSymbols UP-TO-DATE
[        ] > Task :app:validateSigningDebug UP-TO-DATE
[        ] > Task :app:writeDebugAppMetadata UP-TO-DATE
[        ] > Task :app:writeDebugSigningConfigVersions UP-TO-DATE
[ +997 ms] > Task :app:compressDebugAssets
[        ] Execution optimizations have been disabled for task ':app:compressDebugAssets' to ensure correctness due to the following reasons:
[        ]   - Gradle detected a problem with the following location: '/home/joel/source/platformview/build/app/intermediates/assets/debug/mergeDebugAssets'. Reason: Task ':app:compressDebugAssets' uses this output of task
':app:copyFlutterAssetsDebug' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to
https://docs.gradle.org/7.4/userguide/validation_problems.html#implicit_dependency for more details about this problem.
[+2799 ms] > Task :app:packageDebug
[        ] > Task :app:createDebugApkListingFileRedirect UP-TO-DATE
[ +199 ms] > Task :app:assembleDebug
[  +99 ms] Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
[        ] You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
[        ] See https://docs.gradle.org/7.4/userguide/command_line_interface.html#sec:command_line_warnings
[        ] Execution optimizations have been disabled for 1 invalid unit(s) of work during this build to ensure correctness.
[        ] Please consult deprecation warnings for more details.
[        ] BUILD SUCCESSFUL in 35s
[        ] 35 actionable tasks: 7 executed, 28 up-to-date
[ +410 ms] Running Gradle task 'assembleDebug'... (completed in 36.8s)
[  +69 ms] calculateSha: LocalDirectory: '/home/joel/source/platformview/build/app/outputs/flutter-apk'/app.apk
[+1101 ms] ✓  Built build/app/outputs/flutter-apk/app-debug.apk.
[   +4 ms] executing: /home/joel/Android/Sdk/build-tools/31.0.0/aapt dump xmltree /home/joel/source/platformview/build/app/outputs/flutter-apk/app.apk AndroidManifest.xml
[  +25 ms] Exit code 0 from: /home/joel/Android/Sdk/build-tools/31.0.0/aapt dump xmltree /home/joel/source/platformview/build/app/outputs/flutter-apk/app.apk AndroidManifest.xml
[        ] N: android=http://schemas.android.com/apk/res/android
             E: manifest (line=2)
               A: android:versionCode(0x0101021b)=(type 0x10)0x1
               A: android:versionName(0x0101021c)="1.0.0" (Raw: "1.0.0")
               A: android:compileSdkVersion(0x01010572)=(type 0x10)0x1f
               A: android:compileSdkVersionCodename(0x01010573)="12" (Raw: "12")
               A: package="com.example.platformview" (Raw: "com.example.platformview")
               A: platformBuildVersionCode=(type 0x10)0x1f
               A: platformBuildVersionName=(type 0x10)0xc
               E: uses-sdk (line=7)
                 A: android:minSdkVersion(0x0101020c)=(type 0x10)0x13
                 A: android:targetSdkVersion(0x01010270)=(type 0x10)0x1f
               E: uses-permission (line=15)
                 A: android:name(0x01010003)="android.permission.INTERNET" (Raw: "android.permission.INTERNET")
               E: application (line=17)
                 A: android:label(0x01010001)="platformview" (Raw: "platformview")
                 A: android:icon(0x01010002)=@0x7f080000
                 A: android:name(0x01010003)="android.app.Application" (Raw: "android.app.Application")
                 A: android:debuggable(0x0101000f)=(type 0x12)0xffffffff
                 A: android:appComponentFactory(0x0101057a)="androidx.core.app.CoreComponentFactory" (Raw: "androidx.core.app.CoreComponentFactory")
                 E: activity (line=23)
                   A: android:theme(0x01010000)=@0x7f0a0000
                   A: android:name(0x01010003)="com.example.platformview.MainActivity" (Raw: "com.example.platformview.MainActivity")
                   A: android:exported(0x01010010)=(type 0x12)0xffffffff
                   A: android:launchMode(0x0101001d)=(type 0x10)0x1
                   A: android:configChanges(0x0101001f)=(type 0x11)0x40003fb4
                   A: android:windowSoftInputMode(0x0101022b)=(type 0x11)0x10
                   A: android:hardwareAccelerated(0x010102d3)=(type 0x12)0xffffffff
                   E: meta-data (line=38)
                     A: android:name(0x01010003)="io.flutter.embedding.android.NormalTheme" (Raw: "io.flutter.embedding.android.NormalTheme")
                     A: android:resource(0x01010025)=@0x7f0a0001
                   E: intent-filter (line=42)
                     E: action (line=43)
                       A: android:name(0x01010003)="android.intent.action.MAIN" (Raw: "android.intent.action.MAIN")
                     E: category (line=45)
                       A: android:name(0x01010003)="android.intent.category.LAUNCHER" (Raw: "android.intent.category.LAUNCHER")
                 E: meta-data (line=52)
                   A: android:name(0x01010003)="flutterEmbedding" (Raw: "flutterEmbedding")
                   A: android:value(0x01010024)=(type 0x10)0x2
                 E: uses-library (line=56)
                   A: android:name(0x01010003)="androidx.window.extensions" (Raw: "androidx.window.extensions")
                   A: android:required(0x0101028e)=(type 0x12)0x0
                 E: uses-library (line=59)
                   A: android:name(0x01010003)="androidx.window.sidecar" (Raw: "androidx.window.sidecar")
                   A: android:required(0x0101028e)=(type 0x12)0x0
[   +2 ms] Stopping app 'app.apk' on F5121.
[   +1 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell am force-stop com.example.platformview
[ +115 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell pm list packages com.example.platformview
[ +592 ms] package:com.example.platformview
[   +2 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell cat /data/local/tmp/sky.com.example.platformview.sha1
[  +61 ms] 57a5e8147be0902ea81d1627ec14a90fab45a036
[   +1 ms] Installing APK.
[   +2 ms] Installing build/app/outputs/flutter-apk/app.apk...
[        ] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 install -t -r /home/joel/source/platformview/build/app/outputs/flutter-apk/app.apk
[+10707 ms] Performing Streamed Install
                     Success
[        ] Installing build/app/outputs/flutter-apk/app.apk... (completed in 10.7s)
[   +2 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell echo -n 1dd46ee244c28061a288d6fecaeb4b4df16e9b02 > /data/local/tmp/sky.com.example.platformview.sha1
[  +52 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell -x logcat -v time -t 1
[  +71 ms] --------- beginning of main
           06-24 10:21:38.234 I/Finsky  (27957): [6471] kkj.accept(5): IQ: handled package removed
[  +21 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 shell am start -a android.intent.action.RUN -f 0x20000000 --ez enable-dart-profiling true --ez enable-checked-mode true --ez verify-entry-points true
com.example.platformview/com.example.platformview.MainActivity
[ +112 ms] Starting: Intent { act=android.intent.action.RUN flg=0x20000000 cmp=com.example.platformview/.MainActivity (has extras) }
[        ] Waiting for observatory port to be available...
[+2674 ms] Observatory URL on device: http://127.0.0.1:44925/mE437mntM28=/
[   +4 ms] executing: /home/joel/Android/Sdk/platform-tools/adb -s BH9007JH56 forward tcp:0 tcp:44925
[  +27 ms] 38787
[        ] Forwarded host port 38787 to device port 44925 for Observatory
[  +19 ms] Caching compiled dill
[  +66 ms] Connecting to service protocol: http://127.0.0.1:38787/mE437mntM28=/
[ +533 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:38787/mE437mntM28=/.
[ +378 ms] DDS is listening at http://127.0.0.1:40175/AsvzCMIKZuE=/.
[ +125 ms] Successfully connected to service protocol: http://127.0.0.1:38787/mE437mntM28=/
[ +166 ms] DevFS: Creating new filesystem on the device (null)
[ +211 ms] I/zygote64(31667): Do partial code cache collection, code=61KB, data=50KB
[        ] I/zygote64(31667): After code cache collection, code=61KB, data=50KB
[        ] I/zygote64(31667): Increasing code cache capacity to 256KB
[  +57 ms] DevFS: Created new filesystem on the device (file:///data/user/0/com.example.platformview/code_cache/platformviewNRJPMW/platformview/)
[   +3 ms] Updating assets
[ +136 ms] Syncing files to device F5121...
[   +2 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[        ] Processing bundle.
[   +1 ms] <- recompile package:platformview/main.dart 5cdc2b44-879a-45c2-9edf-852b7b5d7536
[        ] <- 5cdc2b44-879a-45c2-9edf-852b7b5d7536
[   +2 ms] Bundle processing done.
[ +121 ms] E/qdutils (31667): Failed to open msm_fb_panel_info node
[  +49 ms] Updating files.
[        ] DevFS: Sync finished
[   +1 ms] Syncing files to device F5121... (completed in 179ms)
[        ] Synced 0.0MB.
[   +2 ms] <- accept
[   +6 ms] Connected to _flutterView/0x730f71de20.
[   +4 ms] Flutter run key commands.
[   +3 ms] r Hot reload. 🔥🔥🔥
[        ] R Hot restart.
[        ] h List all available interactive commands.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] 💪 Running with sound null safety 💪
[        ] An Observatory debugger and profiler on F5121 is available at: http://127.0.0.1:40175/AsvzCMIKZuE=/
[ +305 ms] The Flutter DevTools debugger and profiler on F5121 is available at: http://127.0.0.1:9100?uri=http://127.0.0.1:40175/AsvzCMIKZuE=/
```

```
Analyzing platformview...                                               
No issues found! (ran in 3.0s)
```

```
[✓] Flutter (Channel stable, 3.0.2, on openSUSE Tumbleweed 5.18.4-1-default, locale en_US.UTF-8)
    • Flutter version 3.0.2 at /home/joel/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision cd41fdd495 (2 weeks ago), 2022-06-08 09:52:13 -0700
    • Engine revision f15f824b57
    • Dart version 2.17.3
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/joel/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /home/joel/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /usr/bin/chromium

[✓] Android Studio (version 2021.2)
    • Android Studio at /home/joel/android-studio
    • Flutter plugin version 68.1.2
    • Dart plugin version 212.5744
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] IntelliJ IDEA Community Edition (version 2022.1)
    • IntelliJ at /home/joel/IntelliJ
    • Flutter plugin version 68.1.4
    • Dart plugin version 221.5921.27

[✓] VS Code (version 1.68.1)
    • VS Code at /usr/share/code
    • Flutter extension can be installed from:
      🔨 https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (2 available)
    • F5121 (mobile) • BH9007JH56 • android-arm64  • Android 8.0.0 (API 26)
    • Chrome (web)   • chrome     • web-javascript • Chromium 102.0.5005.115 stable

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

</details>

With transparent background:     |  With solid background
:-------------------------:|:-------------------------:
![Screenshot_20220624-102943](https://user-images.githubusercontent.com/64319336/175496547-ec41d13a-93e8-4c3f-97df-b9958a1b252c.png) | ![Screenshot_20220624-102847](https://user-images.githubusercontent.com/64319336/175496544-86ec5d23-001a-4a40-a3ba-2d8220fb3135.png)

**Comment by joelpaulkoch** [comment]
Thanks for your quick response.
The faulty view was produced on a Sony Xperia X with Android 8.0.0 and I can reproduce it there.
I checked with an old Lenovo phone and there it is shown correctly, just as it is for you.
I'm not sure what to do with that.

**Comment by maheshmnj** [comment]
Thanks for the info @joelpaulkoch, I tried to reproduce this on Xiaomi k20 pro (Android 11) and pixel 4 emulator on macOs and Linux hosts, But I was not able to reproduce the issue. SInce this issue manifests on only certain devices, I am labeling this issue for further investigation from the team.

<img src="https://user-images.githubusercontent.com/31410839/176106653-92f79ab8-c19a-4e67-b082-96f733605bfc.png" width=300>

<details>
<summary>flutter doctor -v (Linux)</summary>

```
[✓] Flutter (Channel stable, 3.0.3, on Ubuntu 20.04.4 LTS 5.13.0-48-generic, locale en_GB.UTF-8)
    • Flutter version 3.0.3 at /home/mahesh/Development/stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 676cefaaff (6 days ago), 2022-06-22 11:34:49 -0700
    • Engine revision ffe7b86a1e
    • Dart version 2.17.5
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/mahesh/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio
    • Flutter plugin version 67.1.2
    • Dart plugin version 212.5744
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.68.0)
    • VS Code at /usr/share/code
    • Flutter extension version 3.42.0

[✓] Connected device (3 available)
    • sdk gphone x86 (mobile) • emulator-5554 • android-x86    • Android 11 (API 30) (emulator)
    • Linux (desktop)         • linux         • linux-x64      • Ubuntu 20.04.4 LTS 5.13.0-48-generic
    • Chrome (web)            • chrome        • web-javascript • Google Chrome 102.0.5005.115

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by stuartmorgan** [comment]
This is the general version of https://github.com/flutter/flutter/issues/104686; see https://github.com/flutter/flutter/issues/104686#issuecomment-1156763554 for more context.

We need to document that for PlatformView use cases where transparency support is critical prior to API 29, plugin developers will need to use Hybrid Composition.

**Comment by joelpaulkoch** [comment]
Could you clarify these two points please:
First, it works absolutely fine on a Lenovo A2016a40 running Android 6.0 (prior to API 29).
Second, I followed the documentation for Hybrid Composition so that's what I used (?).

**Comment by stuartmorgan** [comment]
> First, it works absolutely fine on a Lenovo A2016a40 running Android 6.0 (prior to API 29).

I'm not familiar with the implementation details, so I can't provide any more context here than the comment I linked to above. (Perhaps the behavior for <29 is device-specific?)

> Second, I followed the documentation for Hybrid Composition so that's what I used (?).

It looks like those docs weren't updated when the new mode shipped in 3.0. My understanding is that the standard behavior is now the new mode in most cases, and `initExpensiveAndroidView` switches back to Hybrid Composition.

**Comment by johnmccutchan** [comment]
Closing this stale issue as there is nothing to do:

- You need to be running on Android >= 29 in order to support transparency in texture layer mode.
- You need to use hybrid composition on Android < 29.
